email/reporter_test.go

```diff
@@ -0,0 +1,240 @@
+package email
+
+import (
+	"abuse-scanner/database"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.sia.tech/siad/build"
+)
+
+// TestReporter contains a set of unit tests that cover the reporter struct.
+func TestReporter(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	t.Parallel()
+
+	tests := []struct {
+		name string
+		test func(t *testing.T)
+	}{
+		{
+			name: "ReportMessages",
+			test: testReportMessages,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, test.test)
+	}
+}
+
+// testReportMessages verifies the messages that contain csam get reported
+func testReportMessages(t *testing.T) {
+	t.Parallel()
+
+	// os.Setenv("NCMEC_USERNAME", "[ENTER_USERNAME]")
+	// os.Setenv("NCMEC_PASSWORD", "[ENTER_PASSWORD]")
+	// os.Setenv("NCMEC_DEBUG", "true")
```

Comment:
commented out code

Reply:
I want to leave that in as a placeholder.